src/main/java/uk/nhs/digital/nhsconnect/lab/results/inbound/fhir/EdifactToFhirService.java

```diff
@@ -0,0 +1,28 @@
+package uk.nhs.digital.nhsconnect.lab.results.inbound.fhir;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.hl7.fhir.r4.model.Parameters;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.nhs.digital.nhsconnect.lab.results.inbound.fhir.mapper.FhirTransactionMapper;
+import uk.nhs.digital.nhsconnect.lab.results.inbound.fhir.mapper.NotSupportedFhirTransactionMapper;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.ReferenceTransactionType;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.Transaction;
+
+import java.util.Map;
+
+@Slf4j
+@Component
+@RequiredArgsConstructor(onConstructor_ = @Autowired)
+public class EdifactToFhirService {
```

Comment:
Is this needed for your ticket? I think it's part of NIAD-857

Reply:
Yes, I believe it was a dependency for another file I needed to run the `InboundUserAcceptanceTest.java` test

Reply:
Removed from project following discussion